actions/src/main/java/com/redhat/service/bridge/actions/ActionProvider.java

```diff
@@ -0,0 +1,14 @@
+package com.redhat.service.bridge.actions;
+
+import com.redhat.service.bridge.infra.models.actions.BaseAction;
+import com.redhat.service.bridge.infra.models.dto.ProcessorDTO;
+
+public interface ActionProvider {
+
+    String getType();
```

Comment:
What about moving here the `accept` logic instead of `ActionProviderFactory`?
```suggestion
    String getType();
    
    default boolean accept(String actionType) {
        return getType().equals(actionType);
    }
```